src/app/pages/OpenWalletPage/index.tsx

```diff
@@ -33,9 +44,18 @@ export function SelectOpenMethod() {
         <span>
           <ButtonLink to="private-key" label={t('openWallet.method.privateKey', 'Private key')} primary />
         </span>
-        <span>
-          <ButtonLink to="ledger" label={t('openWallet.method.ledger', 'Ledger')} primary />
-        </span>
+        {webExtensionLedgerAccess ? (
+          <Button
+            style={{ width: 'fit-content' }}
+            onClick={webExtensionLedgerAccess}
+            label={t('ledger.extension.grantAccess', 'Grant access to your Ledger')}
+            primary
+          />
```

Comment:
This flow should only be needed the first time for each unpaired ledger device, not every time. Background page already has the needed permissions.

Reply:
How about on click: attempt to get a device and if it fails openLedgerAccessPopup

Reply:
Can you share more details? Do you want to dispatch saga action first and handle "LedgerNoDeviceSelected" differently or there is some way to get a device and checked if it's paired?

Reply:
I think just `try { await TransportWebUSB.create(); to="ledger" } catch (e) { openLedgerAccessPopup() }`

Reply:
If we want to create usb transport  here we will have to close it in React component as well to avoid `USB Transport error: Failed to execute 'claimInterface' on 'USBDevice': Unable to claim interface.` At this point everything looks fine, but when I will revoke usb access I go through process again and it will end up with `USB Transport error: Failed to execute 'open' on 'USBDevice': The device was disconnected..`.

Reply:
Alright I am able to reproduce this
```tsx
  try {
    await (await TransportWebUSB.create()).close()
    navigate('/open-wallet/ledger')
  } catch (e) {
    webExtensionLedgerAccess()
  }
```

what's very odd is that it works after reloading background page, as if `close()` doesn't fully reset device communication
- start with extension having USB permission
- enumerate accounts **works**, cancel
- remove USB permission
- connect ledger flow, enumerate accounts **does not work**, cancel
- reload background page
- enumerate accounts works, cancel
- remove USB permission
- reload background page
- connect ledger flow, enumerate accounts **works**, cancel